app/src/main/res/values/strings.xml

```diff
@@ -74,4 +74,7 @@
     <string name="fail_to_edit_book">Failed to edit book</string>
     <string name="scanner_scanning">Scanning...</string>
     <string name="accept_this_request">Accept this request?</string>
+    <string name="delete">delete</string>
+    <string name="Add_Edit">Add/Edit</string>
+    <string name="Edit_detail">Edit</string>
```

Comment:
I think we don't need "Add_Edit"? doesn't looks like it is used anywhere, and for snake_case the first letter should be lowercase